packages/cli/src/commands/init/init.ts

```diff
@@ -90,6 +96,20 @@ async function createFromTemplate({
   logger.debug('Initializing new project');
   logger.log(banner);
 
+  let packageManager = pm;
+  if (npm) {
+    logger.warn(
+      'Flag --npm is deprecated and will be removed soon. In the future, please use --pm npm instead.',
+    );
+
+    packageManager = 'npm';
+  }
+
+  const userAgent = userAgentPackageManager();
+  if (userAgent === 'bun') {
+    packageManager = 'bun';
+  }
```

Comment:
this favors bun and introduces some inconsistency in picking the preferred package manger. The algorithm should go like this imho:
1. if flag, set pm from flag
2. if no flag, set pm from userAgent
3. if no userAgent, fallback to default (yarn currently)